src/bournemouth/openrouter.py

```
@@ -48,16 +48,19 @@
 
 
 class ImageUrl(msgspec.Struct, array_like=True):
+    """URL and resolution hint for an image content part."""
```

Ruff (0.11.9)

1 blank line required after class docstring

Insert 1 blank line after class docstring

(D204)